Formula/lib/libayatana-indicator.rb

```diff
@@ -0,0 +1,58 @@
+class LibayatanaIndicator < Formula
+  desc "Ayatana Indicators Shared Library"
+  homepage "https://github.com/AyatanaIndicators/libayatana-indicator"
+  url "https://github.com/AyatanaIndicators/libayatana-indicator/archive/refs/tags/0.9.4.tar.gz"
+  sha256 "a18d3c682e29afd77db24366f8475b26bda22b0e16ff569a2ec71cd6eb4eac95"
+  license "GPL-3.0-or-later"
+
+  # can only be built on Linux until https://github.com/AyatanaIndicators/libayatana-indicator/pull/77 is fixed
+  depends_on "cmake" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "pkgconf" => [:build, :test]
+  depends_on "vala" => :build
+  depends_on "at-spi2-core"
+  depends_on "ayatana-ido"
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "glib"
+  depends_on "gtk+3"
+  depends_on "harfbuzz"
+  depends_on :linux # macOS fatal error: 'gio/gdesktopappinfo.h' file not found
+  depends_on "pango"
+
+  def install
+    args = %w[
+      -DENABLE_IDO=ON
+      -DENABLE_LOADER=ON
+      -DENABLE_TESTS=OFF
+      -DFLAVOUR_GTK3=ON
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    (testpath/"test.c").write <<~C
+      #include <libayatana-indicator/indicator.h>
+      #include <string.h>
+      #include <assert.h>
+      #include <ctype.h>
+      int main() {
+        // Check that INDICATOR_VERSION looks like a semantic version (e.g. X.Y.Z)
+        const char *v = INDICATOR_VERSION;
```

Comment:
You're only testing a macro here, which is a much more limited test than actually testing a library function. (In particular, if you omit linking to the library, you'll see that this test will still succeed.0

Can we do a test that actually requires linking with the library we've just built?

Reply:
Do you have an idea for what method can be tested? I originally tried to test the `get_version` method, but got an error:

- https://github.com/AyatanaIndicators/libayatana-indicator/blob/08c5540e36fbd92d48ea3cc5b4ea4e8731c05a96/src/indicator.h#L30
- https://github.com/Homebrew/homebrew-core/actions/runs/16482534162/job/46600438734#step:4:71

Maybe I missed something simple.